launcher/util.go

```diff
@@ -28,3 +39,68 @@ func FetchImpersonatedToken(ctx context.Context, serviceAccount string, audience
 
 	return []byte(token.AccessToken), nil
 }
+
+// SetTPMDAParams takes in a TPM and updates its Dictionary Attack parameters
+func SetTPMDAParams(tpm io.ReadWriter, maxTries, lockoutInterval, lockoutRecovery uint32) error {
+	// empty auth
```

Comment:
I would have expected this to take TPMDAParams. The parameter names are also inconsistent.

Reply:
done
The naming is inconsistent because seems like there are two sets of DA variables names:
https://github.com/google/go-tpm/blob/364d5f2f78b95ba23e321373466a4d881181b85d/legacy/tpm2/test/tpm2_test.go#L2356

for some reason, I created a mapping:
```
	MaxAuthFail         uint32 // maxTries
	LockoutInterval     uint32 // recoveryTime
	LockoutRecovery     uint32 // lockoutRecovery
```

Reply:
That's just a test. It appears the actual parameters are correct:
https://github.com/google/go-tpm/blob/364d5f2f78b95ba23e321373466a4d881181b85d/legacy/tpm2/tpm2.go#L2297